ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -569,6 +569,7 @@ debug.mode = Debug
 debug.mode.description = Debug mode will print information into the browser console to help developers diagnose problems with the application. After changing this setting reload the app so it applies.
 debug.mode.title = Enable debug mode
 denied = Denied timestamp
+dhis.dataset.monthly.integration = MoH Monthly Integration
```

Comment:
should this be in the config/default translations file instead?

Reply:
Because it references MoH?
Something has to exist here regardless (though perhaps with a more generic name) because the default config only comes into play on fresh instances.

Reply:
Yes I think you're right - moved.